sklearn/decomposition/nmf.py

```diff
@@ -1106,7 +1106,6 @@ def transform(self, X):
             nls_max_iter=self.nls_max_iter, sparseness=self.sparseness,
             beta=self.beta, eta=self.eta)
 
-        self.n_iter_ = n_iter_
```

Comment:
I removed all `self.n_iter` = something in `transform`, but not really sure where this change should be documented. Could you please point me to the right place for that?